operator/model/src/main/java/io/apicurio/registry/operator/api/v1/ApicurioRegistry3SpecKafkaSqlSecurity.java

```diff
@@ -0,0 +1,29 @@
+package io.apicurio.registry.operator.api.v1;
+
+import com.fasterxml.jackson.annotation.*;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.databind.JsonDeserializer.None;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import io.fabric8.kubernetes.api.model.KubernetesResource;
+import lombok.*;
+
+@JsonInclude(Include.NON_NULL)
+@JsonPropertyOrder({ "tls" })
+@JsonDeserialize(using = None.class)
+@NoArgsConstructor
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@Builder
+@Getter
+@Setter
+@ToString
+public class ApicurioRegistry3SpecKafkaSqlSecurity implements KubernetesResource {
```

Comment:
Instead of keep adding suffixes I think we should consolidate on using package/namespaces like we did for Sql:
https://github.com/Apicurio/apicurio-registry/tree/main/operator/model/src/main/java/io/apicurio/registry/operator/api/v1/spec

`ApicurioRegistry3SpecKafkaSqlSecurity` is ways too long of a name ...